### DISCUSSION TITLE
Text wrap in QTable columns

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#11180] [General - Components / Directives / etc] [answered]
How can we wrap text in columns when using QTable ? I've created a sample project here on codepen. I'd like to try and make it so the table extends vertical to avoid the user having to scroll horizontal to view the table content.

https://codepen.io/jokermartini/pen/ZEJXNJw

**Comment by rstoenescu** [comment, marked as answer]
https://quasar.dev/vue-components/table#qtable-api

<img width="654" alt="image" src="https://user-images.githubusercontent.com/4665641/139922914-6fdaf8df-2d13-4670-8584-cc301c639f6e.png">

**Reply by mijaelsaban** [reply]
cool thanks
